### DISCUSSION TITLE
difference between variables for VS button

### DISCUSSION THREAD

**Discussion by JLD95** [flybywiresim/aircraft#7109] [Q&A]
Hi, 
I don't understand the difference between variables for VS button on document https://docs.flybywiresim.com/pilots-corner/a32nx-briefing/a32nx_api/
In order to select vertical speed, the variables <A32NX.FCU_SPD_INC> and <A32NX.FCU_SPD_DEC> don't work for me, I shall use <AP_SPD_VAR_INC> and <AP_SPD_VAR_DEC> which work fine.
I use  X-touch mini and Mobiflight to map these variables.

**Comment by aguther** [comment]
Your post is a bit confusing.

Do you want to manipulate airspeed or vertical speed? Everything hints for vertical speed, but the events you're using are all for airspeed.

The `A32NX.FCU_SPD_INC` and `A32NX.FCU_SPD_DEC` are custom SimConnect events which have been proven to work with SPAD.neXt and FSUIPC. MobiFlight I cannot tell.

**Reply by JLD95** [reply]
Hi,

It is for vertical speed as written in FBW API documentation

![IMG_20220427_190450.jpg](https://user-images.githubusercontent.com/5894146/165580831-8337608e-891a-49a0-9c89-cb6e2a6b57ba.jpg)

**Reply by aguther** [reply]
You wrote:
![image](https://user-images.githubusercontent.com/6629914/165583374-41a29e53-22cc-4597-a805-b6c9d8c2a8d1.png)

**Reply by JLD95** [reply]
I confirm that (A32NX.FCU_VS_INC, A32NX.FCU_VS_DEC) don't work for me:
    {
      "index": 5,
      "event_up": "A32NX.FCU_VS_INC",
      "event_down": "A32NX.FCU_VS_DEC",
      "event_short_press": {"event": "A32NX.FCU_VS_PULL", "type": "manual", "value": 1 },
      "event_long_press": {"event": "A32NX.FCU_VS_PUSH", "type": "manual", "value": 1 },
	    "mobiflightsimvar_led": "(L:A32NX_FCU_VS_MANAGED)"
    },

only (AP_VS_VAR_INC, AP_VS_VAR_DEC) work fine:
    {
      "index": 5,
      "event_up": "AP_VS_VAR_INC",
      "event_down": "AP_VS_VAR_DEC",
	  "alternate_event_up": "AP_VS_VAR_INC",
      "alternate_event_down": "AP_VS_VAR_DEC",
      "event_short_press": {"event": "A32NX.FCU_VS_PULL", "type": "manual", "value": 1 },
      "event_long_press": {"event": "A32NX.FCU_VS_PUSH", "type": "manual", "value": 1 },
	    "mobiflightsimvar_led": "(L:A32NX_FCU_VS_MANAGED)"
    },

**Reply by aguther** [reply]
Cannot reproduce:
https://user-images.githubusercontent.com/6629914/165779599-565ae4e7-1ec4-42f2-87a6-08e55603519f.mp4
